woocommerce/woocommerce-android
# [Support Requests] Remap ticket tags and fields

Summary
==========
Fix issue #8423 by updating Zendesk tags and fields configuration per the guidelines described here: pe5pgL-2k3-p2

```
Mobile App
Form: 360000010286
Tags: woocommerce_mobile_apps, jetpack(legacy from old implementation)
Sub-Category: 25176023: WooCommerce Mobile Apps

Card Reader / IPP
Form: 360000010286
Tags: woocommerce_mobile_apps, product_area_apps_in_person_payments, jetpack(legacy from old implementation)
Sub-Category: 25176023: WooCommerce Mobile Apps

WooCommerce Payments
Form: 189946
Tags: woocommerce_payments, support, payment, product_area_woo_payment_gateway, mobile_app_woo_transfer
Category: 25176003: support
Sub-Category: 25176023: payment
Note: Do not send the woo-mobile-sdk tag.

WooCommerce Plugins
Form: 189946
Tags: woocommerce_core, support, mobile_app_woo_transfer
Category: 25176003: support
Note: Do not send the woo-mobile-sdk tag.

Other Plugins
Form: 189946
Tags: support, store, product_area_woo_extensions, mobile_app_woo_transfer
Category: 25176003: support
Sub-Category: 25176023: store
Note: Do not send the woo-mobile-sdk tag.
```

How to Test
==========
1. Testing will be conducted here pe5pgL-2jR#comment-1912. I recommend focusing on the review of the code implementation, but you can also check the tickets created using the same reference described above.

Update release notes:
